src/SettingsPage/Tabs/Menu.php

```diff
@@ -0,0 +1,33 @@
+<?php declare( strict_types = 1 ); # -*- coding: utf-8 -*-
+
+namespace Adminimize\SettingsPage\Tabs;
+
+class Menu implements TabInterface {
+
+	/**
+	 * Returns the title
+	 *
+	 * @return string
+	 */
+	public function getTitle(): string {
+		return __('Menu');
+	}
+
+	/**
+	 * Returns the subtitle
+	 *
+	 * @return string
+	 */
+	public function getSubTitle(): string {
+		return '';
+	}
+
+	/**
+	 * Returns the tab content
+	 *
+	 * @return string
+	 */
+	public function getContent(): string {
+		return __('<h3>Menu Options</h3>');
```

Comment:
We should exclude the markup. not necessary for an translation, maybe helpful https://gist.github.com/bueltge/49c652187ef10a448bfb